windows/src/developer/TIKE/child/UfrmDebug.pas

```diff
@@ -1124,13 +1147,15 @@ procedure TfrmDebug.SetUIStatus(const Value: TDebugUIStatus);
           StartBatch;
           StatusText := 'Debugging';
           memo.ReadOnly := True;
+          memo.IsDebugging := False;
```

Comment:
Why is this `False` in the duiDebugging case?

Reply:
Yeah, this one is a little tricky. The thing is, while single-step debugging, the control temporarily switches modes so characters can be inserted, etc, if these are a part of the action being debugged. But we don't want the user to make changes to the content of the control while single-step debugging -- if they are on a breakpoint and press Ctrl+X to delete content, then that will mess things up quite badly!